[MST] Add MFA procedure, and content about auth tokens (#1444)
timescale/docs: mst/user-management.md

</procedure>

## Multi-factor user authentication

You can use multi-factor authentication (MFA) to log in to your Managed Service
for TimescaleDB account. This requires an authentication code, provided by the

> Google Authenticator app on your mobile device, to be able to log in to your
> Managed Service for TimescaleDB account.

Reviewer: ```suggestion
Google Authenticator app on your mobile device.
```

Non-blocking suggestion, found this a bit repetitive

You can see which authentication method is in use by each member of your Managed
Service for TimescaleDB project. From the dashboard, navigate to the `Members`
section. Each member is listed in the table with an authentication method of

> either `Password`, or `Two-Factor`.

Reviewer: ```suggestion
either `Password` or `Two-Factor`.
```

Before you begin, install the Google Authenticator app on your mobile device.
For more information, and installation instructions, see
[the Google Authenticator documentation][install-google-authenticator].

<procedure>

### Configuring multi-factor authentication

1.  [Log in to your account][mst-login].
1.  Click the `User information` icon in the top-right of the dashboard to go to
    the `User profile` section.
1.  In the `Authentication` tab, toggle `Two-factor authentication` to
    `Enabled`, and enter your password.
1.  On your mobile device, open the Google Authenticator app, tap `+` and select
    `Scan a QR code`.
1.  On your mobile device, scan the QR code provided by Managed Service for
    TimescaleDB.
1.  In your Managed Service for TimescaleDB dashboard, enter the confirmation
    code provided by the Google Authenticator app, and click
    `Enable Two-Factor Auth`.

</procedure>

<highlight type="warning">

> If you have lost access to the mobile device uses for your multi-factor

Reviewer: ```suggestion
If you have lose access to the mobile device you use for multi-factor
```

authentication, you cannot sign in to your Managed Service for TimescaleDB
account. To regain access to your account, on the login screen, click
`Forgot password?` and follow the step to reset your password. When you have
regained access to your account, reconfigure multi-factor authentication.
</highlight>

## User authentication tokens

Every time a registered user logs in, Managed Service for TimescaleDB creates a
new authentication token. This occurs for login events using the portal, and
using the API. By default, authentication tokens expire after 30 days, but the
expiry date is adjusted every time the token is used. This means that tokens can

> be used indefinitely, if the user logs in at least every thirty days.

Reviewer: Not terribly bothered by whether we spell out `thirty` or write `30`, but we should make it consistent within this paragraph.

Author: oof, good spot!

```suggestion
be used indefinitely, if the user logs in at least every 30 days.
```

for TimescaleDB dashboard. [Log in to your account][mst-login], and click the
`User information` icon in the top-right of the dashboard to go to the
`User profile` section. In the `Authentication` tab, the table lists all current
authentication tokens.

When you make authentication changes, such as enabling two factor authentication
or resetting a password, all existing tokens are revoked. In some cases, a new
token is immediately created so that the web console session remains valid. You
can also manually revoke authentication tokens from the `User profile` page
individually, or click `Revoke all tokens` to revoke all current tokens.

Additionally, you can click `Generate token` to create a new token. When you
generate a token on this page, you can provide a description, maximum age, and
an extension policy. Generating authentication tokens in this way allows you to
use them with monitoring applications that make automatic API calls to Managed
Service for TimescaleDB.

<highlight type="important">
There is a limit to how many valid authentication tokens are allowed per user.
This limit is different for tokens that are created as a result of a sign in

> operation, and for tokens created explicitly. For automatically-created tokens,

Reviewer: ```suggestion
operation, and for tokens created explicitly. For automatically created tokens,
```

> the system automatically deletes the oldest tokens as new ones are creates. For

Reviewer: ```suggestion
the system automatically deletes the oldest tokens as new ones are created. For
```

> explicitly-created tokens, older tokens are not deleted unless they expire or

Reviewer: ```suggestion
explicitly created tokens, older tokens are not deleted unless they expire or
```

> are manually revoked. This can result in explicitly-created tokens that stop

Reviewer: ```suggestion
are manually revoked. This can result in explicitly created tokens that stop
```
